### ISSUE TITLE
[DataCap Application] <Fogmeta Labs> - <Meta Ark>

### ISSUE THREAD

**Issue by hengdingy** [filecoin-project/filecoin-plus-large-datasets#1968]
### Data Owner Name

Fogmeta Labs

### Data Owner Country/Region

China

### Data Owner Industry

Web3 / Crypto

### Website

https://fogmetalabs.com/metaark

### Social Media

```text
Twitter: https://twitter.com/FogMeta
GitHub: https://github.com/FogMeta
```

### Total amount of DataCap being requested

5PiB

### Weekly allocation of DataCap requested

1PiB

### On-chain address for first allocation

f1m5ysspdkkjhq2hb5sogit3j5snypexjqks5g6gi

### Data Type of Application

Private Commercial/Enterprise

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
The research of Fogmeta Labs spans multiple levels from system technology, infrastructure, and middleware to services and solutions, and involves future systems, network technology and business, distributed systems and management, information management, and interactive and innovative services. Based on the views on and practices in the industry, Fogmeta also solves the problem of business complexity through operations optimization and other technologies.

Meta Ark is a data onboarding service provided by Fogmeta Labs for helping clients store their corporate datasets onto IPFS/Filecoin. Please refer to: https://fogmetalabs.com/metaark.
```

### Is this project associated with other projects/ecosystem stakeholders?

Yes

### If answered yes, what are the other projects/ecosystem stakeholders

```text
Meta Ark is supported by FilSwan(https://www.filswan.com/).
```

### Describe the data being stored onto Filecoin

```text
These are private corporate datasets owned by our Meta Ark clients. Meta Ark is a data onboarding service provided by Fogmeta Labs for helping clients store their corporate datasets onto IPFS/Filecoin. Please refer to: https://fogmetalabs.com/metaark.
```

### Where was the data currently stored in this dataset sourced from

Other

### If you answered "Other" in the previous question, enter the details here

```text
Meta Ark is a data onboarding product developed by Fogmeta Labs. And the datasets come from our Meta Ark clients.
```

### How do you plan to prepare the dataset

others/custom tool

### If you answered "other/custom tool" in the previous question, enter the details here

```text
We're going to use the `go-mc-sdk` to prepare the datasets. https://github.com/FogMeta/go-mc-sdk
```

### Please share a sample of the data

```text
We apologize, but due to confidentiality reasons, we are unable to disclose or share the private data here.
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [ ] I confirm

### If you chose not to confirm, what was the reason

```text
These are private corporate data owned by our clients.
```

### What is the expected retrieval frequency for this data

Monthly

### For how long do you plan to keep this dataset stored on Filecoin

2 to 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, Africa, North America, South America, Europe, Australia (continent), Antarctica

### How will you be distributing your data to storage providers

HTTP or FTP server, IPFS, Shipping hard drives, Others

### How do you plan to choose storage providers

Slack, Partners, Others

### If you answered "Others" in the previous question, what is the tool or platform you plan to use

```text
We plan to use FilSwan Platform (https://console.filswan.com/dashboard#/dashboard) for choosing storage providers who meet our requirements.
```

### If you already have a list of storage providers to work with, fill out their names and provider IDs below

```text
The storage providers we'd like to work with are presented below. Some of them are from the FilSwan Platform.
f03624
f010088
f02301
f08399
f02401
f0187709
f01163272
f01402814
f01072221
f0240185
f0143858
f01390330
f01225882
f0717969
f03223
f01395673
f01786736
f0836160
f032824
f01443744
f01871352
f01907556
f01946551
f02012951
f01970630
```

### How do you plan to make deals to your storage providers

Others/custom tool

### If you answered "Others/custom tool" in the previous question, enter the details here

```text
https://github.com/FogMeta/go-mc-sdk
```

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by Sunnyiscoming** [comment]
### Datacap Request Trigger
**Total DataCap requested**
> 5PiB

**Expected weekly DataCap usage rate**
> 1PiB

**Client address**
>f1m5ysspdkkjhq2hb5sogit3j5snypexjqks5g6gi

**Comment by hengdingy** [comment]
@kevzak    Can you help us confirm that fogmeta has completed KYC, and KYB via Synaps check and also submitted the registration form including dataset size, sample, and SP details?  the verification proof has been sent to you in slack.

**Comment by kevzak** [comment]
> @kevzak Sorry it was a mistake. I am aware I cannot sign E-Fil applications. Please remove my signature as I cannot do it on the UI.

We've removed the signature, thanks for the update.

**Comment by newwebgroup** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacec4q2ywdq3ffjpelvmp37hz4r6urib7zttkagsvem7rvwyyvzlvqw 
#### Address 
> f1m5ysspdkkjhq2hb5sogit3j5snypexjqks5g6gi
#### Datacap Allocated
> 256.00TiB
#### Signer Address
> f1e77zuityhvvw6u2t6tb5qlnsegy2s67qs4lbbbq
#### Id
> 82da0d22-ffb5-4c0b-a31f-b58d284c0f46
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacec4q2ywdq3ffjpelvmp37hz4r6urib7zttkagsvem7rvwyyvzlvqw

**Comment by newwebgroup** [comment]
@raghavrmadya  @kevzak 
Sorry, I signed the wrong LDN when I was approving Fil-E.
Because I signed **Approve**, I can't withdraw it at the operation desk.
Please withdraw the approved DC

**Comment by kevzak** [comment]
@hengdingy please do not move forward with deals using this DataCap. This was signed in error. cc @raghavrmadya

**Comment by raghavrmadya** [comment]
Flagged for DC removal

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
@kevzak @raghavrmadya  The datacap of this address has **never** been used. We are actively engaged in project development and preparation, and will have communicated with the efil+ team again

**Comment by kevzak** [comment]
Hi @hengdingy the datacap was allocated by two separate notaries in error. It's normal process for it to be removed and you can get approval properly once your application is ready. 

This application still has not passed E-Fil Requirements without proof of clients. Any updates?

**Comment by kevzak** [comment]
Also, we have a new KYC ID check available on [filplus.storage](https://filplus.storage/). Please complete when you have a chance as a requirement for E-Fil+ to add additional layer of trust and to verify this GitHub account. See details: [LINK](https://medium.com/filecoin-plus/implementation-of-a-third-party-kyc-id-check-for-fil-applicants-e5e825b1484e)

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
yes, it is alive

**Comment by Normalnoise** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by Normalnoise** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by Normalnoise** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
keep alive

**Comment by hengdingy** [comment]
@kevzak Fogmeta's Efil+ Verification has been Approved,
<img width="834" alt="image" src="https://github.com/filecoin-project/filecoin-plus-large-datasets/assets/102578774/75776350-0177-4d4b-b309-fa69e3d6e953">
 and Meta-Ark has done many improvements, everybody can use it by Web and SDK:
 - WEB: https://fogmetalabs.com/metaark
 - SDK: https://github.com/fogmeta/go-mc-sdk
 
 and Meta-Ark SDK has supported to rebuild the data from the filecoin storage, can we start using the datacap?

**Comment by kevzak** [comment]
@hengdingy if I recall - you were supposed to have a meeting with Filecoin Foundation, Paul W, to review clients. [LINK](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1968#issuecomment-1591402176). Any update? This project is on hold until then.

**Comment by hengdingy** [comment]
Thank @kevzak , Meta-Ark is always improving the features, for now, it is available to all people by the [web](https://fogmetalabs.com/metaark), and our client, [multichain.storage](https://multichain.storage), is using the [SDK](https://github.com/fogmeta/go-mc-sdk) to backup the data.  if needed we can provide the client's information.

**Comment by Normalnoise** [comment]
checker:manualTrigger

**Comment by Normalnoise** [comment]
checker:manualTrigger

**Comment by kevzak** [comment]
> Thank @kevzak , Meta-Ark is always improving the features, for now, it is available to all people by the [web](https://fogmetalabs.com/metaark), and our client, [multichain.storage](https://multichain.storage), is using the [SDK](https://github.com/fogmeta/go-mc-sdk) to backup the data. if needed we can provide the client's information.

@hengdingy, Yes we need to see some detail about:
- Clients
- Clients Data
- Size of Dataset
- Geo Distribution of copies

**Comment by kevzak** [comment]
I would prefer someone from Filecoin Foundation to review the details and sign off on this before we trigger this application. Could be Yan Bo

**Comment by hengdingy** [comment]
Thanks @kevzak , Meta-ark is a WEB3 data storage and data rebuild product based on the Filecoin network. It can serve the different Clients, our clients include [Multichain.storage](https://multichain.storage), [Lagrange](https://lagrangedao.org) and other company or organizasions. All data from the Clients' data will be backed up to the Filecoin network by the Meta-Ark.

So more details are as follows:

* Clients: [Multichain.storage](https://multichain.storage), [Lagrange](https://lagrangedao.org) 
* Client's Data: User’s data, product data, cache data etc.
* Proof of Size of Dataset: see the pictures about 200TiB data, which is a part of the IPFS clusters.
* Geo Distribution of copies: Every files will be stored to the SPs around the world, and we will keep at least 4 Active copies in the network.

![image](https://github.com/filecoin-project/filecoin-plus-large-datasets/assets/107467232/f1b81758-cb54-4a23-949c-226f07e26dc2)
![image](https://github.com/filecoin-project/filecoin-plus-large-datasets/assets/107467232/3ee5a209-74fd-4e05-8c2e-5688222ff3bc)
![image](https://github.com/filecoin-project/filecoin-plus-large-datasets/assets/107467232/bfebb983-12d9-43d2-8d6a-941aee013995)

welcome communities review our application. And @kevzak, I will DM Filecoin Foundation member to help us review our case.

**Comment by Normalnoise** [comment]
keep live

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by Sunnyiscoming** [comment]
Please provide ID, City, Country, Organization of each SP here.

**Comment by hengdingy** [comment]
> Please provide ID, City, Country, Organization of each SP here.
@Sunnyiscoming 
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/2286#issuecomment-1835787739

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
checker:manualTrigger

**Comment by hengdingy** [comment]
> ## DataCap and CID Checker Report[1](#user-content-fn-1-1db7edf0335f7232847266ee87111be6)
> No active deals found for this client.
> 
> ## Footnotes
> 1. To manually trigger this report, add a comment with text `checker:manualTrigger` [↩](#user-content-fnref-1-1db7edf0335f7232847266ee87111be6)

checker:manualTrigger

**Comment by Kirameta** [comment]
manualTrigger

**Comment by hengdingy** [comment]
manualTrigger

**Comment by hengdingy** [comment]
manualTrigger

**Comment by hengdingy** [comment]
> @hengdingy this pathway is no longer active to receive DataCap. If you are interested, you can apply here for Enterprise Datacap. https://github.com/fidlabs/Enterprise-Data-Pathway/issues/new/choose

What's the Project ID?

**Comment by kevzak** [comment]
Not sure I understand your question @hengdingy
